src/libraries/System.Runtime.Extensions/tests/System/StringComparer.cs

```diff
@@ -86,8 +86,10 @@ public static IEnumerable<object[]> UpperLowerCasing_TestData()
             //                          lower                upper          Culture
             yield return new object[] { "abcd",             "ABCD",         "en-US" };
             yield return new object[] { "latin i",          "LATIN I",      "en-US" };
-            yield return new object[] { "turky \u0131",     "TURKY I",      "tr-TR" };
-            yield return new object[] { "turky i",          "TURKY \u0130", "tr-TR" };
+            if (PlatformDetection.IsThreadingSupported){
```

Comment:
Same here: How is globalization coupled with threading?

Reply:
This needs some kind of ICU detection.  The problem is the tests are failing when invariant mode is enabled.  

@safern is working on disabling / enabling overall.  Perhaps we should remove the change from this PR for the time being.

Reply:
Yeah, as port of: https://github.com/dotnet/runtime/pull/38976 I’m adding a PlatformDetection property to know if we’re running in Invariant so probably we should skip this test if that’s the case.

Reply:
I merged my PR in case you want to use that property to skip these cases.